mono/metadata/threadpool-worker-default.c

```diff
@@ -194,19 +192,17 @@ COUNTER_READ (void)
 	return counter;
 }
 
-static gpointer
-rand_create (void)
-{
-	mono_rand_open ();
-	return mono_rand_init (NULL, 0);
-}
-
 static guint32
-rand_next (gpointer *handle, guint32 min, guint32 max)
+rand_next (guint32 min, guint32 max)
 {
 	ERROR_DECL (error);
-	guint32 val;
-	mono_rand_try_get_uint32 (handle, &val, min, max, error);
```

Comment:
It looks like the whole thing was needed just for TP, can we remove mono_rand_try_get_uint32 + rest?